### PR TITLE
Fix player controls spacing and sizing

### PR DESCRIPTION
**Pull request by PickleNik**
Before:
image
After:
image

Motivation behind this: 
image
image

### CODE REVIEW

**Comment by PickleNik** [comment]
@zt64 will merg

File: app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt [hunk 1]
```diff
@@ -45,26 +45,30 @@ fun PlayerControls(
...
+                    modifier = Modifier.size(30.dp),
                     imageVector = Icons.Default.SkipPrevious,
                     contentDescription = stringResource(R.string.skip_previous)
                 )
             }
 
-            IconButton(onClick = onClickPlayPause) {
+            IconButton(
+                modifier = Modifier.
+                then(Modifier.size(64.dp)),
```
Comment (zt64) [inline app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt:61]: No need for `.then`, you can just set the size directly

**Review by zt64 — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt (hunk 1 above)
Comment (PickleNik) [reply app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt:61]: okay, thank, i was copying stackoverflow cuz icon wasn't going above 40.dp on its own

File: app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt [hunk 2]
```diff
@@ -45,26 +45,29 @@ fun PlayerControls(
...
+                    modifier = Modifier.size(30.dp),
                     imageVector = Icons.Default.SkipPrevious,
                     contentDescription = stringResource(R.string.skip_previous)
                 )
             }
 
-            IconButton(onClick = onClickPlayPause) {
+            IconButton(
+                modifier = Modifier.size(64.dp),
+                onClick = onClickPlayPause) {
```
Comment (zt64) [inline app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt:61]: This `) {` should be carried on to the next line like how it is everywhere else

**Review by zt64 — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt (hunk 2 above)
Comment (PickleNik) [reply app/src/main/kotlin/com/hyperion/ui/component/PlayerControls.kt]: bet

**Review by zt64 — APPROVED** [review]

**Pull request merged**